maven_install_2.12.json

```diff
@@ -1,6 +1,6 @@
 {
     "dependency_tree": {
-        "__AUTOGENERATED_FILE_DO_NOT_MODIFY_THIS_FILE_MANUALLY": -429398716,
+        "__AUTOGENERATED_FILE_DO_NOT_MODIFY_THIS_FILE_MANUALLY": -810309042,
```

Comment:
I just repinned those but apparently the 1.18 branches haven’t been modified sufficiently long that this turned into a huge change. not much we can do about this.